README.md

```diff
@@ -437,6 +438,17 @@ repository.records.first.sets
 
 Return an `Array` of `String` [set specs](#fieldhandsetspec) indicating set memberships of this record.
 
+#### `Fieldhand::Record#dump`
```

Comment:
thats a good suggestion, but since we have a metadata method, that kinda threw me off, as it would not be in symmetry with `to_xml`. Maybe we should align both those methods somehow?